src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java

```diff
@@ -153,4 +163,15 @@ public boolean isFooterVisible() {
       return false;
     }
   }
+
+  public boolean areHubLinksVisible() {
+    try {
+      return gamesHub.isDisplayed()
+          && moviesHub.isDisplayed()
+          && tvHub.isDisplayed();
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)